### PR TITLE
New icons for Application Preferences list

### PR DESCRIPTION
**Pull request by agrogers**
I forgot to get your latest changes so there are a couple of lines in the settings.xml that conflict. I tried to fix by starting a new branch but failed :(

### CODE REVIEW

**Comment by agrogers** [comment]
Oh, i can fix them right in the github interface. I think it is ok now.

File: app/src/main/res/xml/settings.xml [hunk 1]
```diff
@@ -97,11 +97,13 @@
 			android:title="@string/hide_window_buttons_title"
 			android:summary="@string/hide_window_buttons_summary"
 			android:defaultValue="false"
+			android:icon="@drawable/ic_hide_window_button_bar_24dp"/>
 			/>
 		<SwitchPreferenceCompat android:key="hide_bible_reference_overlay"
 			android:title="@string/hide_bible_reference_overlay_title"
 			android:summary="@string/hide_bible_reference_overlay_summary"
-			android:defaultValue="false"
+			android:icon="@drawable/ic_hide_bible_reference_overlay_24dp"/>
```
Comment (tuomas2) [inline app/src/main/res/xml/settings.xml]:
```suggestion
			android:icon="@drawable/ic_hide_bible_reference_overlay_24dp"
```

File: app/src/main/res/xml/settings.xml [hunk 2]
```diff
@@ -97,11 +97,13 @@
 			android:title="@string/hide_window_buttons_title"
 			android:summary="@string/hide_window_buttons_summary"
 			android:defaultValue="false"
+			android:icon="@drawable/ic_hide_window_button_bar_24dp"/>
```
Comment (tuomas2) [inline app/src/main/res/xml/settings.xml]:
```suggestion
			android:icon="@drawable/ic_hide_window_button_bar_24dp"
```

File: app/src/main/res/xml/settings.xml [hunk 3]
```diff
@@ -74,7 +74,7 @@
 			android:summary="@string/prefs_night_mode_summary"
 			android:entries="@array/prefs_night_mode_descriptions"
 			android:entryValues="@array/prefs_night_mode_values"
-			android:icon="@drawable/ic_night_mode_24"
+			android:icon="@drawable/ic_night_mode_switching_24dp"/>
```
Comment (tuomas2) [inline app/src/main/res/xml/settings.xml]:
```suggestion
			android:icon="@drawable/ic_night_mode_switching_24dp"
```

File: app/src/main/res/xml/settings.xml [hunk 4]
```diff
@@ -53,7 +53,7 @@
 		<SwitchPreferenceCompat android:key="auto_fullscreen_pref"
 			android:title="@string/auto_fullscreen"
 			android:summary="@string/auto_fullscreen_summary"
-			android:icon="@drawable/ic_full_screen_24"
+			android:icon="@drawable/ic_full_screen_by_scrolling_24dp"/>
```
Comment (tuomas2) [inline app/src/main/res/xml/settings.xml]:
```suggestion
			android:icon="@drawable/ic_full_screen_by_scrolling_24dp"
```

**Pull request merged**